### PR TITLE
Add C++ Implementation for Jury Marks - Intro Sets

### PR DESCRIPTION
**Pull request by tachithanhdanh**

### CODE REVIEW

File: solutions/CF Jury Marks.mdx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+author: Danh Ta Chi Thanh
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/53302)
+
+<LanguageSection>
+
+<CPPSection>
+
+**Time Complexity:** $\mathcal{O}(K^2 \log K)$
```
Comment (HarryWangATX) [inline solutions/CF Jury Marks.mdx:14]: Complexity should be outside

**Review by HarryWangATX — CHANGES_REQUESTED** [review]

File: solutions/CF Jury Marks.mdx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		a[i] += a[i-1];
+	}
+	b.resize(N); 
+	for (int& p : b) cin >> p;
+	set<int> init;
+	for (int m = 1; m <= K; ++m) {
+		init.insert(b.front() - a[m]);
+	}
+	int ans = 0;
+	for (const int& d : init) { //must be `const int` (or `auto` for simplicity)
```
Comment (bqi343) [inline solutions/CF Jury Marks.mdx:40]: or just `int d: init`?

File: solutions/CF Jury Marks.mdx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+		set<int> points;
+		for (int i = 1; i <= K; ++i) {
+			points.insert(d + a[i]);
+		}
+		bool flag = true;
+		for (int& p : b) flag &= points.count(p);
```
Comment (bqi343) [inline solutions/CF Jury Marks.mdx:46]: same here

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: solutions/CF Jury Marks.mdx (hunk 2 above)
Comment (tachithanhdanh) [reply solutions/CF Jury Marks.mdx:40]: I used `int d : init` and it got a compilation error.

Comment (tachithanhdanh) [reply solutions/CF Jury Marks.mdx:40]: > or just `int d: init`?

https://codeforces.com/contest/831/submission/107320598

Comment (bqi343) [reply solutions/CF Jury Marks.mdx:40]: https://ideone.com/q1HAIz

File: solutions/CF Jury Marks.mdx (hunk 3 above)
Comment (tachithanhdanh) [reply solutions/CF Jury Marks.mdx:46]: You want me to delete the `&` operator?

File: solutions/CF Jury Marks.mdx (hunk 2 above)
Comment (tachithanhdanh) [reply solutions/CF Jury Marks.mdx]: I've just changed as you said and it seemed fine. 
https://codeforces.com/contest/831/submission/107377082

File: solutions/CF Jury Marks.mdx (hunk 1 above)
Comment (tachithanhdanh) [reply solutions/CF Jury Marks.mdx]: Is this time complexity fine?
I also think that O(K*(K*logK + K*N)) is suitable.

File: solutions/CF Jury Marks.mdx (hunk 3 above)
Comment (bqi343) [reply solutions/CF Jury Marks.mdx]: Yea, do you know what it does?

https://www.learncpp.com/cpp-tutorial/references/
https://www.learncpp.com/cpp-tutorial/references-and-const/

File: solutions/CF Jury Marks.mdx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	ios_base::sync_with_stdio(0);
+	cin.tie(0);
+	int K, N;
+	cin >> K >> N;
+	vector<int> a, b;
```
Comment (bqi343) [inline solutions/CF Jury Marks.mdx:27]: write `vector<int> a(K+1), b(N);` and remove the resizes

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: solutions/CF Jury Marks.mdx (hunk 4 above)
Comment (tachithanhdanh) [reply solutions/CF Jury Marks.mdx:27]: sorry, it's my bad habit :/

**Review by bqi343 — APPROVED** [review]

**Review by Farrius — APPROVED** [review]

**Pull request merged**